packages/blockchain-wallet-v4-frontend/src/scenes/plugin/CoinsList/Header.tsx

```diff
@@ -56,7 +58,7 @@ const Header = (props: any) => {
         {balance}
       </Text>
       <Flex justifyContent='center' gap={24}>
-        <ButtonWrapper>
+        <ButtonWrapper to='/plugin/receive'>
```

Comment:
probably a better name should be used here since it's still a link. Maybe `ButtonLink` or `ButtonLinkWrapper` or even `LinkWrapper`. Think about it please))